modules/apps/portal-workflow/portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/internal/runtime/integration/test/WorkflowInstanceManagerImplTest.java

```diff
@@ -158,6 +160,51 @@ public void testSearchWorkflowInstancesWhenThereIsAnUnregisteredHandler()
 			workflowInstances.toString(), 0, workflowInstances.size());
 	}
 
+	@Test
+	public void testSearchWorkflowInstancesWhenTwoUsersSubmitAnEntry()
+		throws Exception {
+
+		WorkflowDefinition workflowDefinition =
+			_workflowDefinitionManager.deployWorkflowDefinition(
+				TestPropsValues.getCompanyId(), TestPropsValues.getUserId(),
+				RandomTestUtil.randomString(), RandomTestUtil.randomString(),
+				FileUtil.getBytes(
+					getResourceInputStream(
+						"single-approver-workflow-definition.xml")));
+
```

Comment:
Hey @MarinhoFeliphe, you don't need to deploy a new single approver definition. It is enough to use the existent one, like you did in the `testSearchWorkflowInstancesWhenThereIsAnUnregisteredHandler`.

Thx